knative-operator/pkg/common/openshift.go

```diff
@@ -71,6 +75,72 @@ func ingress(ks *servingv1alpha1.KnativeServing, c client.Client) error {
 	return nil
 }
 
+// Update updates Knative controller env to use cluster wide proxy information
+func Update(ks *servingv1alpha1.KnativeServing, c client.Client) error {
+	var proxyEnv = map[string]string{
+		"HTTP_PROXY":  "HTTP_PROXY",
+		"HTTPS_PROXY": "HTTPS_PROXY",
+		"NO_PROXY":    "NO_PROXY",
+	}
+	operatorNS, err := k8sutil.GetOperatorNamespace()
+	if err != nil {
+		return err
+	}
+	podData, err := k8sutil.GetPod(context.TODO(), c, operatorNS)
```

Comment:
I don't quite understand why we need to fetch the pod here? 🤔 Can we not just do `os.Getenv("HTTP_PROXY")` etc. to get the same info?

Reply:
Yes we can use i realized it after your comment
Thank you